vunit/ui.py

```diff
@@ -393,6 +394,36 @@ def add_external_library(self, library_name, path, vhdl_standard=None):
         self._project.add_library(library_name, abspath(path), vhdl_standard, is_external=True)
         return self.library(library_name)
 
+    def add_csv(self, project_csv_path, vhdl_standard=None):
+        """
+        Add a project configuration, mapping all the libraries and files
```

Comment:
The description does not describe the format of the csv file or the meaning of relative paths of the file names.

Reply:
I have some WIP related to a similar approach with JSON:

- https://github.com/1138-4EB/vunit/tree/json_cfg_run

where the 'generic' `run-py` contains:

- https://github.com/1138-4EB/vunit/blob/f67be68b6e8a114a69f1ac67dba92b6f79de11c2/examples/vhdl/array_axis_vcs/cfg_run.py#L16-L20
- https://github.com/1138-4EB/vunit/blob/f67be68b6e8a114a69f1ac67dba92b6f79de11c2/examples/vhdl/array_axis_vcs/cfg_run.py#L35-L48

and the example JSON itself is:

- https://github.com/1138-4EB/vunit/commit/f67be68b6e8a114a69f1ac67dba92b6f79de11c2#diff-c8cec296822f0068205fef2fce6117ac

I'd be glad if the format/approach was explicitly documented, in order to follow the same pattern when I return to it.